test/schema/ListPublicEvents.schema.js

```diff
@@ -0,0 +1,67 @@
+const listPublicEventsSchema = {
+  type: 'object',
+  required: ['status', 'body'],
+  properties: {
+    status: { type: 'number' },
+    body: {
+      type: 'array',
+      properties: {
+        id: { type: 'number' },
+        type: { type: 'string' },
```

Comment:
how can you improve the verification for this property? is this property more than a string? how can you validate the specific set of values?

Reply:
With an enum keyword perhaps